eventq_aws/lib/eventq_aws/aws_eventq_client.rb

```diff
@@ -44,16 +44,23 @@ def raise_event(event_type, event)
             subject: event_type
           )
         end
+        Eventq.log(:debug) do
```

Comment:
@vaughanbrittonsage suggested this so that the code is never evaluated in production